src/um/control.rs

```diff
@@ -0,0 +1,562 @@
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::guiddef::{GUID};
+use shared::wtypes::{BSTR};
+use um::oaidl::{IDispatch, IDispatchVtbl, LPDISPATCH, VARIANT};
+use um::unknwnbase::{IUnknown, IUnknownVtbl, LPUNKNOWN};
+use um::winnt::{HRESULT};
+pub type LONG_PTR = i64;
+RIDL!{#[uuid(0x56a868b9, 0x0ad4, 0x11ce, 0xb0, 0x3a, 0x00, 0x20, 0xaf, 0x0b, 0xa7, 0x70)]
+interface IAMCollection(IAMCollectionVtbl): IDispatch(IDispatchVtbl) {
+    fn get_Count(
+        plCount: *mut i32,
+    ) -> HRESULT,
+    fn Item(
+        lItem: i32,
+        ppUnk: *mut LPUNKNOWN,
+    ) -> HRESULT,
+    fn get__NewEnum(
+        ppUnk: *mut LPUNKNOWN,
+    ) -> HRESULT,
+}}
+RIDL!{#[uuid(0x56a868b1, 0x0ad4, 0x11ce, 0xb0, 0x3a, 0x00, 0x20, 0xaf, 0x0b, 0xa7, 0x70)]
+interface IMediaControl(IMediaControlVtbl): IDispatch(IDispatchVtbl) {
+    fn Run(
+    ) -> HRESULT,
```

Comment:
```suggestion
    fn Run() -> HRESULT,
```
Zero argument functions should be a single line.